### ISSUE TITLE
[Question] Is this docker image still working fine with meteor 1.7.0.1?

### ISSUE THREAD

**Issue by WayneUong** [zodern/meteor-docker#4]
@zodern

**Comment by zodern** [comment]
Yes. Have you had any problems with it?

**Comment by WayneUong** [comment]
@zodern This is the error log I got. The node version is wrong (8.11.2 is the right one) and app won't start.

```
=> Setting node version
NODE_VERSION=8.8.1
v8.8.1 is already installed.
Now using node v8.8.1 (npm v5.4.2)
Now using node v8.8.1 (npm v5.4.2)
=> Starting meteor app on port 80
Note: you are using a pure-JavaScript implementation of bcrypt.
While this implementation will work correctly, it is known to be
approximately three times slower than the native implementation.
In order to use the native implementation instead, run

  meteor npm install --save bcrypt

in the root directory of your application.
2018-06-11T23:39:39.464Z - info: &setup variables
2018-06-11T23:39:39.469Z - info: &migrate db
{"line":"80","file":"migrations_server.js","message":"Migrations: Not migrating, already at version 21","time":{"$date":1528760380325},"level":"info"}
2018-06-11T23:39:40.413Z - info: &start cron
2018-06-11T23:39:40.417Z - info: &restart main server
events.js:182
      throw er; // Unhandled 'error' event
      ^

Error: listen EACCES 0.0.0.0:80
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at Server.setupListenHandle [as _listen2] (net.js:1337:19)
    at listenInCluster (net.js:1395:12)
    at doListen (net.js:1504:7)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)

```

**Comment by WayneUong** [comment]
It deploys fine with `abernix/meteord` btw.

**Comment by zodern** [comment]
Version 0.4.0 is now available with Node 8.11.2.
